pre_install_report/library/pre_install_check_permissions.py

```diff
@@ -42,6 +45,9 @@
 PROVISIONER_AZURE_DISK = "kubernetes.io/azure-disk"
 PROVISIONER_AWS_EBS = "kubernetes.io/aws-ebs"
 
+INGRESS_REL_EQUAL = '==1.18'
+INGRESS_REL_LESS = '<1.18'
```

Comment:
this shouldn't be necessary, if they are not >=1.18 then the opposite will be available without needing this defined